controllers/controllers.go

```diff
@@ -0,0 +1,11 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// HealthCheckHandler indicates that the server is up and running.
+func HealthCheckHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm not really familiar with golang, so apologies if I'm misinterpreting this, but it looks like you are passing in a parameter (`r *http.Request`) that isn't being used.

Reply:
@dav3r `mux.HandleFunc()` in line 12 in `main.go` requires a handler function with those two parameters, `http.ResponseWriter` and `http.Request` to be passed in.

Per the go module documentation:
```go
func (*http.ServeMux).HandleFunc(pattern string, handler func(http.ResponseWriter, *http.Request))
// HandleFunc registers the handler function for the given pattern.
```

Reply:
Got it - appreciate the info, thanks!